packages/editor/src/components/index.js

```diff
@@ -22,7 +22,6 @@ export {
 	default as RichText,
 	RichTextShortcut,
 	RichTextToolbarButton,
-	RichTextInserterItem,
```

Comment:
Is this a breaking change? Should we stub it at least to avoid breakage or find a way to move it automatically to the toolbar?

Reply:
Yeah, mapping to `RichTextToolbarButton` should do the trick.